### PR TITLE
Basic layout of navbar

### LINKED ISSUE #13: Create a NavBar component
For now we want to be able to go to the following pages: `Home`, `Events`, `Contact`, `About`.
We want to make it transparent and display the links on desktop.

### PR DESCRIPTION
**Pull request by ZestyScrubz**
Closes #13 
## Overview
Interactable navigation bar.
- Logo goes to home page
- About goes to /about

Responsive Navbar not working
- Hamburger icon showing on desktop

Use this tutorial
https://www.youtube.com/watch?v=qUV2kgYdr9k&t=772s&pp=ygUoaG93IHRvIG1ha2UgYSBuYXZiYXIgZm9yIG1vYmlsZSB0YWlsd2luZA%3D%3D

### CODE REVIEW

File: app/components/NavBar.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                                <li className="p-3 hover:text-indigo-200 rounded-md transition-all cursor-pointer">Contact</li>
+                            </ul>
+                        </div>
+
+                        {/* Hamburger Icon (only visible on smaller screens) not working */}
+                        
+                        <i
+                            className="bx bx-menu xl:hidden block" 
+                            onClick={() => setIsMenuOpen(!isMenuOpen)} 
+                        ></i>
```
Comment (Cheekane) [inline app/components/NavBar.tsx:42]: Change to an Icon image instead.

File: app/components/NavBar.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                            <li className="p-3 hover:text-indigo-200 rounded-md transition-all cursor-pointer">
+                                <a href="/event" className="text-white hover:text-indigo-200">Events</a>
+                            </li>
+                        </ul>
+                        
+                        <div className="relative hidden md:flex items-center justify-center gap-3 font-semibold text-base">
+                            <ul>
+                                <li className="p-3 hover:text-indigo-200 rounded-md transition-all cursor-pointer">Contact</li>
+                            </ul>
+                        </div>
```
Comment (Cheekane) [inline app/components/NavBar.tsx:35]: Put this with the other links.

File: app/page.tsx [hunk 3]
```diff
@@ -1,101 +1,11 @@
...
-            width={16}
-            height={16}
-          />
-          Go to nextjs.org →
-        </a>
-      </footer>
-    </div>
+    <main>
+      <NavBar />
+      <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'></link>
```
Comment (Cheekane) [inline app/page.tsx:8]: Don't use this.

**Review by Cheekane — CHANGES_REQUESTED** [review]
@ZestyScrubz Here are some of your first requested changes.
1. Shorten the NavBar (it's too tall)
2. Align the dropdown menu
3. Make it mobile compatible

File: app/components/NavBar.tsx (hunk 1 above)
Comment (ZestyScrubz) [reply app/components/NavBar.tsx]: I used <svg>

File: app/components/NavBar.tsx (hunk 2 above)
Comment (ZestyScrubz) [reply app/components/NavBar.tsx]: thumbs up

File: app/page.tsx (hunk 3 above)
Comment (ZestyScrubz) [reply app/page.tsx:8]: thumbs up

**Comment by ZestyScrubz** [comment]
## Overview
Fixed hamburger menu
Navbar link not centered because of logo

**Review by Cheekane — APPROVED** [review]

**Pull request closed without merging**